### ISSUE TITLE
Reconsider positioning of UI on profile page

### ISSUE THREAD

**Issue by kraftner** [wecodemore/wcm-avatar#31]
I am not sure it is reasonable that the Avatar UI on the profile page is on the very top, somewhat hackishly implemented via the `all_admin_notices`hook. `show_user_profile` and `edit_user_profile` might be more reasonable choices.

Good news: Just switching the hook to one of these two works flawlessly. Once #26 is done this might be an easy fix.

**Comment by kraftner** [comment]
Okay I take that "flawlessly" back because using one of those other hooks ends up with nested `<form>` tags. Maybe this was the reason it was done like this in the first place.

**Comment by franz-josef-kaiser** [comment]
@kraftner Yes, that was the reason – which I forgot, but now that you mention it …

Maybe a quick PR with a note for the README? Documenting "design decisions" in the form of a _note to our later selfs_ might not be too stupid. Helped me in some cases with other projects.

**Comment by kraftner** [comment]
Yeah for now adding this to the docs probably is fine.

But it should still be possible to switch the position, it just means we need to piggyback on the main form instead of rolling our own.
